transpile.cf.js

```diff
@@ -1,38 +1,47 @@
 import fs from 'fs'
 import path from 'path'
 
-const empty = x => fs.readdirSync(x).forEach(f => fs.unlinkSync(path.join(x, f)))
-    , ensureEmpty = x => !fs.existsSync(x) ? fs.mkdirSync(x) : empty(x)
+const empty = (x) =>
```

Comment:
linter

Reply:
Are you sure you're using the .eslint from this repo?